editor/animation_bezier_editor.h

```diff
@@ -130,17 +130,8 @@ class AnimationBezierTrackEdit : public Control {
 	Vector2 moving_handle_right;
 	int moving_handle_mode = 0; // value from Animation::HandleMode
 
-	struct PairHasher {
-		static _FORCE_INLINE_ uint32_t hash(const Pair<int, int> &p_value) {
-			int32_t hash = 23;
-			hash = hash * 31 * hash_one_uint64(p_value.first);
-			hash = hash * 31 * hash_one_uint64(p_value.second);
-			return hash;
-		}
-	};
-
-	HashMap<Pair<int, int>, Vector2, PairHasher> additional_moving_handle_lefts;
-	HashMap<Pair<int, int>, Vector2, PairHasher> additional_moving_handle_rights;
+	HashMap<Pair<int, int>, Vector2> additional_moving_handle_lefts;
```

Comment:
This change will change the hashing behavior, but I guess that's fine?

Reply:
It seems like that at a glance, but I've simply moved the hashing function to `hashfuncs.h` instead, which means pairs/keyvalues are hashed the same as before but via the default implementation.

Reply:
Are you sure? Your new implementation makes sense to me, but the old one hashes the ints as `uint64_t` and combines them by multiplying by 31.
Edit: Oh, I think your new implementation is from `PairHash` from `pair.h` while the one used previously in this file is from `PairHasher` from directly above. I can see how this is confusing 😄